crates/bevy_internal/Cargo.toml

```
@@ -190,14 +190,15 @@ bevy_ci_testing = ["bevy_dev_tools/bevy_ci_testing", "bevy_render?/ci_limits"]
 animation = ["bevy_animation", "bevy_gltf?/bevy_animation"]
 
 bevy_sprite = ["dep:bevy_sprite", "bevy_gizmos?/bevy_sprite", "bevy_image"]
-bevy_pbr = ["dep:bevy_pbr", "bevy_gizmos?/bevy_pbr", "bevy_image"]
+bevy_pbr = ["dep:bevy_pbr", "bevy_gizmos?/bevy_pbr", "bevy_light"]
```

was bevy_image supposed to be here?

---

bevy_light depends on bevy_camera which depends on bevy_mesh which depends on bevy_light, so its in the chain. but im realizing its enough to just enable bevy_render instead, so im gonna do that
```suggestion
bevy_pbr = ["dep:bevy_pbr", "bevy_gizmos?/bevy_pbr", "bevy_render"]
```

---

actually wait. this is wrong: bevy_render does not depend on bevy_light. i need both here